chat/message/src/MessageBanner.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
```

Comment:
this file should be inside `/MessageBanner`

Reply:
Moved this + moved `Message.stories.tsx` under `/Message`